static/webpack-config.js

```diff
@@ -60,7 +60,16 @@ module.exports = function () {
     new RemovePlugin({
       after: {
         root: `${jamboConfig.dirs.output}`,
-        include: ['static'],
+        test: [
+          {
+            folder: 'static',
+            method: absoluteFilePath => {
+              const filePathRelativeToOutput = path.relative(jamboConfig.dirs.output, absoluteFilePath);
+              return !filePathRelativeToOutput.startsWith('static/assets');
```

Comment:
Can you expand on the reasoning behind this for my own edification?

Reply:
instead of erasing the entire static folder, it's only erasing files that don't start with "static/assets"
the path.relative is because the plugin sends the absolute file path to this hook, e.g
`/Users/oshi/answers-hitchhiker-theme/test-site/public/static/assets/slapshot.png`

I tried just adding exclude: ['static/assets'] but for whatever reason that doesn't work with `include`, they even say in their example for exclude that you have to use `test` instead of `include` if you want to use `exclude`. From my testing it looks like exclude only works with individual files and not folders though, so I had to resort to this weird checking.

https://www.npmjs.com/package/remove-files-webpack-plugin

Reply:
Ah, got it. It just took me a second to follow the logic here. Would this explanatory value be accurate/helpful?

```suggestion
              const isFileWithinStaticAssetOutputDir = filePathRelativeToOutput.startsWith('static/assets')
              return !isFileWithinStaticAssetOutputDir;
```